lte/gateway/docker/README.md

```diff
@@ -51,3 +51,15 @@ on a VM. However we are not there yet as the containerized AGW currently depends
 on a patched Open vSwitch installation on the host machine. The magma VM happens
 to have the right packages installed, and thus can currently be used as a quick
 and dirty way to run the containers locally.
+
+### Running the S1AP integration tests against the containerized AGW
+
+At present, of the S1AP integration tests, it has been verified that `test_attach_detach` can be run versus the containerized AGW.
+Before running this test, one should execute the following command to set a required environment variable:
+
+```
+sed -i 's/MAGMA_DEV_MODE=0/MAGMA_DEV_MODE=1/g' $MAGMA_ROOT/lte/gateway/docker/.env
```

Comment:
Is this possible after the images are created, i.e., during runtime?

Reply:
We tried various things but nothing seemed to work. For example, you can parse an environment variable as part of a `docker compose` command, but not as part of the deprecated `docker-compose` command we're using. We thought we might try to migrate this to an override file, for example, but seeing as this was just an attempt to get the first test running, we thought this might be okay for the moment.

Reply:
I realise you were asking a difference question. I've updates the docs with the sentence below stating that you need to restart the containers.